pyproject.toml

```diff
@@ -32,6 +32,11 @@ directory = "translation"
 name = "Translations"
 showcontent = true
 
+[[tool.towncrier.type]]
+directory = "api"
+name = "API changes"
+showcontent = true
+
```

Comment:
I was tempted to put the changelog in "misc", but that is supposed to be hidden. Also i don't want to mix those with user facing changes.

Reply:
The problem I have with 'api' is when I hear it, I assume something that *is* user-facing. I'm not sure what a better word would be, to describe "changes to CLI toolkit/design that make it easier to add new commands", tho :(